packages/world/src/systems/ForceFieldSystem.sol

```diff
@@ -140,8 +143,10 @@ contract ForceFieldSystem is System {
     caller.activate();
     caller.requireAdjacentToFragment(fragmentCoord);
 
-    ObjectType objectType = EntityObjectType._get(forceField);
-    require(objectType == ObjectTypes.ForceField, "Invalid object type");
+    {
```

Comment:
why does this cause a stack issue if no new variables were added?

Reply:
ah I think I can remove, it was before when I was doing the program calls inlined, but now they are in a separate function.